examples/run/alpaka/seeding_example_alpaka.cpp

```diff
@@ -63,8 +67,15 @@ int seq_run(const traccc::opts::track_seeding& seeding_opts,
     using Idx = uint32_t;
 
     using Acc = ::alpaka::ExampleDefaultAcc<Dim, Idx>;
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+    ::sycl::queue q;
+    vecmem::sycl::queue_wrapper qw{&q};
+    traccc::alpaka::vecmem::host_device_types<
+        ::alpaka::trait::AccToTag<Acc>::type>::device_copy copy(qw);
```

Comment:
Technically you don't **have to** create a queue_wrapper object like this. You could also do:

```suggestion
    traccc::alpaka::vecmem::host_device_types<
        ::alpaka::trait::AccToTag<Acc>::type>::device_copy copy(&q);
```

And yeah... I'll be interested in your ideas on how to abstract this in a better way... 🤔 (I only have some vague ideas at the moment.)